### PR TITLE
Completed prep-css-flexbox.

### PR DESCRIPTION
**Pull request by ChorongShin**
Screen Shot 2021-11-11 at 11 18 39 PM
Screen Shot 2021-11-11 at 11 18 56 PM
Screen Shot 2021-11-11 at 11 20 02 PM
Screen Shot 2021-11-11 at 11 20 26 PM

### CODE REVIEW

File: prep-css-flexbox/align-items.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - align-items</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="starwars.css" />
+    <style>
+      #card-deck {
+        display: flex;
+        flex-basis: calc(100% / 3);
```
Comment (thebearingedge) [inline prep-css-flexbox/align-items.html]: Careful `flex-basis` goes on flex items (children of elements with `display: flex`). You can add another rule set with a selector for `.card-container` and apply the `flex-basis` there.

File: prep-css-flexbox/flex-wrap.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - flex-wrap</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="starwars.css" />
+    <style>
+      #card-deck {
+        display: flex;
+        flex-wrap: wrap;
+        flex-basis: calc(100% / 6);
```
Comment (thebearingedge) [inline prep-css-flexbox/flex-wrap.html]: Same here, regarding the application of `flex-basis`, try moving it to your rule set for `.card-container` elements.

File: prep-css-flexbox/justify-content.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - justify-content</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="starwars.css" />
+    <style>
+      #card-deck {
+        display: flex;
+        flex-basis: 25%;
```
Comment (thebearingedge) [inline prep-css-flexbox/justify-content.html]: Here as well.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by ChorongShin** [comment]
Screen Shot 2021-11-12 at 4 11 04 PM
Screen Shot 2021-11-12 at 4 13 57 PM
Screen Shot 2021-11-12 at 4 16 19 PM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**